po/zh-CN.po

```diff
@@ -1619,6 +1628,12 @@ msgid ""
 "analyzer.github.io/manual.html#vimneovim), and many others. There is also a "
 "different IDE available called [RustRover](https://www.jetbrains.com/rust/)."
 msgstr ""
+"安装Rust之后，你应当配置你的编辑器或IDE以开始使用Rust。大多数编辑器使用[rust-"
+"analyzer](https://rust-analyzer.github.io/)以达成此目的。它为[VS Code]"
+"(https://code.visualstudio.com/)、[Emacs](https://rust-analyzer.github.io/"
+"manual.html#emacs)、[Vim/Neovim](https://rust-analyzer.github.io/manual."
+"html#vimneovim)及其他许多编辑器提供了自动补全及定义跳转的功能。同样也可以用一"
+"个叫[RustRover](https://www.jetbrains.com/rust/)的特别IDE。"
```

Comment:
I thought the word "special" as the alternative of "different" because I had not find a suitable translation yet. I will remove this word unless I find a suitable one.